.github/workflows/python-package.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10"]
```

Let's add at least 3.11 for now: 

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```